src/commands/fun/leaderboard.js

```diff
@@ -7,39 +7,46 @@ const Balance = require("../../schemas/balance");
 module.exports = {
     data: new SlashCommandBuilder()
         .setName("leaderboard")
-        .setDescription("Gives the amount of points a user has!"),
+        .setDescription("Displays the top 10 users with wizard points!"),
     async execute(interaction) {
         Balance.find()
             .sort({ balance: -1 })
-            .limit(10)
+            .limit(20)
```

Comment:
We may want to look at creating a "Balance Archive" document (Mongo equivalent of table) to move users into when the relevant event is fired.
This way, we keep the limit to 10, don't require the extra overhead of another loop & conditionals, but still retain their information for when/if they rejoin.

Doesn't have to be implemented straight away, but definitely a consideration. Happy to support you developing this yourself if you would prefer.